testApp/Startup.cs

```diff
@@ -31,8 +31,9 @@ public void ConfigureServices(IServiceCollection services)
                 options.MinimumSameSitePolicy = SameSiteMode.None;
             });
 
+            services.AddMvc(option => option.EnableEndpointRouting = false) ;
 
-            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+            // services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);

```

Comment:
Drop this comment out as it serves no purpose now :+1:

Reply:
Good catch - done